docs/pages/Workspace.md

```diff
@@ -0,0 +1,125 @@
+# UI5 Workspaces
+
+!!! warning
+    **UI5 Workspaces is a feature currently in development.** It is not part of any UI5 Tooling release and therefore cannot be used yet.  
+
+!!! example
+    ```yaml title="ui5-workspace.yaml"
+    specVersion: workspace/1.0
+    metadata:
+        name: default
+    dependencyManagement:
+   	    resolutions:
+       	    - path: ../heavy.library
+    ```
+
+## General Concept
+
+UI5 Workspaces can be used to create a personalized local development environment for a UI5 project. They allow to use UI5 dependencies from local directories without the need to use the link features of package managers like npm. "UI5 dependencies" generally refers to projects that have a `ui5.yaml`.
+
+Workspaces are typically configured in a `ui5-workspace.yaml` file, located next to the project's `ui5.yaml`. The file can contain one or many workspace configurations, each separated by [three dashes](https://yaml.org/spec/1.2.2/#22-structures).
+
+If a workspace configuration named `default` exists, it will be used automatically; otherwise the workspace must be specified using the UI5 CLI parameter `--workspace`.
+
+Workspaces can only be used in the project that is currently being worked on, i.e. the current root project. Workspace configurations of dependencies are ignored.
+
+This concept has been discussed in an RFC: [RFC 0006 Local Dependency Resolution](https://github.com/SAP/ui5-tooling/blob/rfc-local-dependency-resolution/rfcs/0006-local-dependency-resolution.md)
+
+
+## Configuration
+A UI5 Workspace configuration must define a specification version (`specVersion`) compatible with its configuration. For more information, see [Workspace Specification Versions](#workspace-specification-versions).
+
+```yaml
+specVersion: "workspace/1.0"
+```
+
+### Metadata
```

Comment:
here we use uppercase headline "Metadata", below lowercase "name". we should align this

Reply:
This is aligned with Configuration.md: https://sap.github.io/ui5-tooling/stable/pages/Configuration/#general-configuration

Only single properties are lowercase, all others (typically grouping multiple properties) are treated as "headlines" and written uppercase